flake: agent/agentssh: panic on MacOS

Seen here: https://github.com/coder/coder/actions/runs/15590571046/job/43908338608?pr=18329


```
=== FAIL: agent/agentssh  (0.00s)
panic: runtime error: unsafe.Slice: ptr is nil and len is not zero

goroutine 1 [running]:
github.com/shirou/gopsutil/v4/cpu.perCPUTimes(0x140004b2900)
	/Users/runner/work/_temp/go-mod-cache/github.com/shirou/gopsutil/v4@v4.25.3/cpu/cpu_darwin.go:158 +0x4dc
github.com/shirou/gopsutil/v4/cpu.TimesWithContext({0x0?, 0x0?}, 0x1)
	/Users/runner/work/_temp/go-mod-cache/github.com/shirou/gopsutil/v4@v4.25.3/cpu/cpu_darwin.go:70 +0x70
github.com/shirou/gopsutil/v4/cpu.Times(...)
	/Users/runner/work/_temp/go-mod-cache/github.com/shirou/gopsutil/v4@v4.25.3/cpu/cpu_darwin.go:59
github.com/shirou/gopsutil/v4/cpu.init.0()
	/Users/runner/work/_temp/go-mod-cache/github.com/shirou/gopsutil/v4@v4.25.3/cpu/cpu.go:65 +0xf0
FAIL	github.com/coder/coder/v2/agent/agentssh	0.043s
```

--- a/go.mod
+++ b/go.mod
@@ -170,7 +170,7 @@ require (
 	github.com/prometheus/common v0.63.0
 	github.com/quasilyte/go-ruleguard/dsl v0.3.22
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/shirou/gopsutil/v4 v4.25.3
+	github.com/shirou/gopsutil/v4 v4.25.4
 	github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
 	github.com/spf13/afero v1.14.0
 	github.com/spf13/pflag v1.0.6

